webapp/src/Controller/API/TeamController.php

```diff
@@ -49,6 +49,14 @@ public function __construct(
         parent::__construct($entityManager, $dj, $config, $eventLogService);
     }
 
+    private function mask_ranking(?Response $normalResponse, string $entity="team"): ?Response
```

Comment:
Instead of using a function let's inline the check. It's basically the same amount of code and makes it more clear what is happening.

Reply:
So duplicate the code in this case?

Reply:
It's basically one statement that needs to be duplicated, so I prefer the clarity on what's actually happening over reading another function

Reply:
This is now changed.